packages/classes/src/config.ts

```diff
@@ -31,7 +31,8 @@ const DEFAULTS: CrafterConfig = {
     GET_NAV_TREE: '/api/1/site/navigation/tree.json',
     GET_BREADCRUMB: '/api/1/site/navigation/breadcrumb.json',
     TRANSFORM_URL: '/api/1/site/url/transform.json',
-    SEARCH: 'crafter-search/api/2/search/search.json'
+    SEARCH: 'crafter-search/api/2/search/search.json',
+    ENGINE_SEARCH: 'api/1/site/elasticsearch/search'    
```

Comment:
Please rename to `Elasticsearch`. Make sure it's one word. Like this: `ELASTICSEARCH`.

Are you sure you have the right URL?

Reply:
I checked with Jose R. and it seems fine